### ISSUE TITLE
I need help with a fine tuned accent

### ISSUE THREAD

**Issue by GUUser91** [Plachtaa/seed-vc#134]
@Plachtaa I'm trying to train a model with a fine tuned british accent. I  gathered about 14 minutes of audio for the dataset. I set max steps and max epochs to 1000. But the output file doesn't a british accent. Here's the input file.
https://vocaroo.com/1gY2BK8MTiTF
Here's the reference file.
https://vocaroo.com/1bTycJauDp9i
Output file from a pretrained model
https://vocaroo.com/1cdag3gy5958
Output file from a fine tuned model
https://vocaroo.com/187AFfhPXbAn

**Comment by Plachtaa** [comment]
Hi there,
seed-vc only ennables timbre conversion, there is no guarantee for accent conversion
hope this clarifies
thx
